docs/src/components/code-block/editor.tsx

```diff
@@ -1,49 +1,49 @@
 import Theme, { useTheme } from '@pluralsight/ps-design-system-theme'
 
 import cx from 'classnames'
-import Highlight, { Language, defaultProps } from 'prism-react-renderer'
-import React, { HTMLAttributes } from 'react'
+import Highlight, { defaultProps } from 'prism-react-renderer'
+import React, { HTMLAttributes, useContext } from 'react'
 
+import { CodeBlockContext } from './index'
 import styles from './styles.module.css'
 import { darkTheme, lightTheme } from './theme'
 
 interface EditorProps extends HTMLAttributes<HTMLPreElement> {
```

Comment:
if we aren't going to spread the props on the child node then we can remove the `extends HTMLAttributes<HTMLPreElement>`